autoshop/autoshop/middleware.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from analytics.models import AnalyticsRequestsRecord
+
+
+class AnalyticsMiddleware:
+
+    def __init__(self, response):
+        self.response = response
+
+    def __call__(self, request):
+        response = self.response(request)
+        request_type = request.method
+        request_date = datetime.datetime.now()
+        response_status = response.status_code
+        if response_status == 200:
```

Comment:
Lets create a `utils.py` file in analytics and add some method there to populate the analytics table according to request or response.